client/models/article.js

```diff
@@ -0,0 +1,12 @@
+import _s from 'underscore.string'
+import { pluck } from 'underscore'
+
+export const getArticleByline = (article) => {
+  const { contributing_authors, author } = article
+
+  if (contributing_authors && contributing_authors.length) {
+    return _s.toSentence(pluck(contributing_authors, 'name'))
+  } else if (author.name) {
+    return author.name
+  }
+}
```

Comment:
Ah yeah, this is was I was thinking about, sorry didn't see it until now. I guess if we aren't using the other methods more than once, it's fine to keep the logic inline. But I do like having this as a home for this kind of business logic 👍

Reply:
Does this seem like an appropriate place to store these files?  Wondering if it is correct to call it a model if it is not really a class but just a collection of utils files.